fic_modules/markdown_modules.py

```diff
@@ -178,8 +182,10 @@ def write_date_header(file_name, datetime_object):
     file.write("\n" + base_table + date_header + "\n")
     file.close()
     if LOGGER:
-        print("Generated date header for file:", file_name, " with datestamp",
-              str(datetime.utcnow()))
+        LOGGER.info("Generated date header for file:" + str(file_name) +
+                    " with datestamp" + str(datetime.utcnow()))
+        # print("Generated date header for file:", file_name, " with datestamp",
```

Comment:
If we don't require this anymore, it should be removed.

Reply:
Fixed.